website/docs/dev/ide-setup.md

```diff
@@ -9,7 +9,7 @@ for the development of Fluss itself.
 
 :::note
 Whenever something is not working in your IDE, try with the Maven
-command line first (`mvn clean package -DskipTests`) as it might be your IDE
+command line first (`./mvnw clean package -DskipTests`) as it might be your IDE
```

Comment:
I prefer to use the native `mvn` command in documentation for users familiar with maven, and add a note here to instruct users to use `./mvnw` to replace `mvn` and explain maven wrapper ensures the correct Maven version (3.8.6).